pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.8"
 odin = "^2.9"
-django-oscar = {version = "^3.2.2", allow-prereleases = true}
+django-oscar = {version = "^3.2.3a1", allow-prereleases = true}
```

Comment:
I recommend using `"^3.2"`.

When you relock the poetry file this will resolve to the latest version that is `>=3.2.0 <4.0.0`

Reply:
This won't resolve to 3.2.3a1, which is necessary in this case

Reply:
Just checked and confirmed it does lock to 3.2.3a1 with:
`{version = "^3.2", allow-prereleases = true}`

Reply:
With `{version = "^3.2", allow-prereleases = true}`
I am getting version 3.2.2, not sure why that is.
<img width="512" alt="image" src="https://github.com/django-oscar/django-oscar-odin/assets/88422175/76579978-c708-4119-9278-dd1b2ee23414">